indexer/ai/src/main/java/com/google/cloudsearch/AISkillDriver.java

```diff
@@ -0,0 +1,42 @@
+package com.google.cloudsearch;
+
+import com.google.cloudsearch.ai.AISkill;
+import com.google.cloudsearch.ai.AISkillSet;
+import com.google.cloudsearch.exceptions.InvalidConfigException;
+import com.google.common.collect.Multimap;
+import org.json.simple.JSONObject;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class AISkillDriver {
+    private AISkillSet skillSet;
+    private Boolean isInvalidConfig = false;
+    private static Logger log = Logger.getLogger(AISkillDriver.class.getName());
+
+    public AISkillDriver(JSONObject aiConfig, JSONObject schema) {
+        try{
+            this.skillSet = new AISkillSet(aiConfig, schema);
+        }
+        catch(InvalidConfigException e){
+            this.isInvalidConfig = true;
+            log.warning("AI Skill configuration is invalid. Structured data won't be populated");
+            e.printStackTrace();
+        }
```

Comment:
why are we catching this exception and continuing ?
If the config is invalid, let the exception bubble up to the user

Reply:
If the config is invalid, then the corresponding structured data fields won't be populated, and this should not interrupt the rest of the warning of the connector, so I just thought of warning the user and not throwing the error.

Reply:
we should halt the whole indexing if configuration is invalid

Reply:
Okay, I will throw the error instead.